app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -217,7 +216,9 @@ class SuggestedEditsTasksFragment : Fragment() {
                     } else if (!maybeSetDisabledStatus(80)) { // TODO: use edit quality metric from API response
 
                         editQualityStatsView.setGoodnessState((editQualityState++) % 7) // TODO: use edit quality metric from API response
-                        // TODO: apply edit streak from API response
+
+                        editStreakStatsView.setTitle(resources.getQuantityString(R.plurals.suggested_edits_edit_streak_detail_text,
+                                editorTaskCounts.editStreak, editorTaskCounts.editStreak))
```

Comment:
Just curious, when will it show `editorTaskCounts.getLastEditDate()`?

Reply:
Good call, and... done.